### PR TITLE
email-common.profile: allow bsfilter

### PR DESCRIPTION
**Pull request by marek22k**

### CODE REVIEW

**Review by kmk3 — COMMENTED** [review]
What is the project page for the program that creates this path?

For clarity, change the commit message title to the PR title and add the
project page to the commit message body.

Other than that, LGTM.

**Comment by marek22k** [comment]
> What is the project page for the program that creates this path?

https://bsfilter.org/
https://duckduckgo.com/?t=ftsa&q=bsfilter+spam
https://directory.fsf.org/wiki/Bsfilter

**Review by kmk3 — APPROVED** [review]

File: etc/profile-a-l/email-common.profile [hunk 1]
```diff
@@ -8,6 +8,7 @@ include email-common.local
 #include globals.local
 
 noblacklist ${HOME}/.bogofilter
+noblacklist ${HOME}/.bsfilter
```
Comment (glitsj16) [inline etc/profile-a-l/email-common.profile:11]: A `noblacklist foo` only makes sense when there's a `blacklist foo`, like in the ${HOME}/.bogofilter case:

https://github.com/netblue30/firejail/blob/1f5f84f9596ba7d26c2f2e35dad7728e12afaec9/etc/inc/disable-programs.inc#L53

This protects the configuration from apps that don' need access to it.

Please add the needed `blacklist ${HOME}/.bsfilter` to `disable-programs.inc` (respecting sort order).

**Review by glitsj16 — CHANGES_REQUESTED** [review]
Add blacklist counterpart.

**Review by glitsj16 — APPROVED** [review]
LGTM

**Comment by netblue30** [comment]
merged, thanks!

**Pull request merged**